### PR TITLE
docs: add how-tos and the base of the quickstart

### PR DESCRIPTION
**Pull request by MaferMazu**
## Description

This PR adds:
- How-tos: Create an Open edX Application for the Plugin Settings
- How-tos: Create enrollment requests manually
- The base of the Quickstart

## Additional information

image
Compile without warnings

image
Index page

image
How-tos: Create an Open edX Application for the Plugin Settings

image
How-tos: Create enrollment requests manually

image
The base of the Quickstart

## Testing

- Create a python8 venv and activate it
```
virtualenv -p python3.8 venv
source venv/bin/activate
```
- Install requeriments
```
make requirements
```
- Enter in docs
```
cd docs
```
- Compile
```
make html
```
- Run the server
```
cd ..
make serve_docs
```
(By default uses port 8000)
or
```
cd ..
python -m http.server 8200 --directory build/html
```
(If you need to serve this in another port)

### CODE REVIEW

**Comment by MaferMazu** [comment]
@wlvasquez, can you help me review this PR of documentation? Particularly the texts I added.

File: docs/source/how-tos/create_enrollment_requests_manually.rst [hunk 1]
```diff
@@ -1,2 +1,42 @@
...
+    .. image:: /_images/how-tos/create_enroll_request/new_enroll_request.png
+        :alt: New Enrollment Request
+
+* Use a valid course ID that will be used in the request. e.g. course-v1:edX+DemoX+Demo_Course.
+
+* Enter the email of the user who will be used in the request.
+
+* Select a Course Mode that the course has.
+
+* Select the type of request you want to execute. If you select Enroll, you will create an enrollment, and if you select Un-enroll, you will set a soft unenrollment (enrollment with status inactive).
```
Comment (wlvasquez) [inline docs/source/how-tos/create_enrollment_requests_manually.rst:32]: Enroll / Unenroll should be in bold, more noticable.

**Review by wlvasquez — APPROVED** [review]
A few suggestions and comments.

**Review by julianramirez2 — DISMISSED** [review]
Please consider @wlvasquez's recommendation; everything else seems correct to me.

File: docs/source/how-tos/create_enrollment_requests_manually.rst (hunk 1 above)
Comment (MaferMazu) [reply docs/source/how-tos/create_enrollment_requests_manually.rst]: Thanks Wanda :D

**Review by wlvasquez — APPROVED** [review]
Ok

**Review by julianramirez2 — APPROVED** [review]

**Pull request merged**